npm/vite-dev-server/src/startServer.ts

```diff
@@ -59,7 +59,9 @@ const resolveServerConfig = async ({ viteConfig, options }: StartDevServerOption
   // Ask vite to pre-optimize all dependencies of the specs
   finalConfig.optimizeDeps = finalConfig.optimizeDeps || {}
 
-  finalConfig.optimizeDeps.entries = [...options.specs.map((spec) => spec.relative), supportFile]
+  if (options.specs.length || supportFile) {
```

Comment:
Should probably make sure that if specs is undefined, make sure that we don't crash.

Reply:
OK I'll add this protection

Reply:
Done 7b012b2